### PR TITLE
 Remove TxTimlock from the mempool

### PR DESCRIPTION
**Pull request by majecty**
I updated the PR #12.

### CODE REVIEW

File: core/src/miner/mem_pool.rs [hunk 1]
```diff
@@ -601,61 +587,13 @@ impl MemPool {
 
     /// Checks the timelock of transactions starting from `start_seq`.
```
Comment (sgkim126) [inline core/src/miner/mem_pool.rs]: I think we should change the comment.

File: core/src/miner/mem_pool.rs [hunk 2]
```diff
@@ -601,61 +587,13 @@ impl MemPool {
...
-    /// If there's no transaction which should wait timelock, returns None.
-    fn check_new_transactions(
-        &self,
-        public: Public,
-        seqs: &[u64],
-        next_seq: u64,
-        current_block_number: PoolingInstant,
-        current_timestamp: u64,
-    ) -> Option<u64> {
+    fn check_transactions(&self, public: Public, start_seq: u64) -> u64 {
```
Comment (sgkim126) [inline core/src/miner/mem_pool.rs]: And I think renaming this method to `next_seq_of_queued` would be good.

**Comment by majecty** [comment]
@sgkim126 I updated this PR.

**Comment by majecty** [comment]
I fixed conflicts.

**Pull request merged**